tasks/app-design-offline.md

```diff
@@ -5,44 +5,44 @@
 
 # Offline Web Applications (optional)
 
-> Note: this task is optional because most likely, you will not use knowledge obtained from it during our course. But, you will need this information when you will work as a developer. At least a general understanding of the concept. That's why finishing this task is up to you.
+> Примітка: це завдання є необов’язковим, оскільки, швидше за все, ви не будете використовувати знання, отримані з нього під час нашого курсу. Але ця інформація вам знадобиться, коли ви будете працювати розробником. Принаймні загальне розуміння понять. Тому робити це завдання чи ні - is up to you.
 
 [Offline Web Applications](https://www.udacity.com/course/offline-web-applications--ud899)
 
-If you decide to complete the task then
-do the following after you're done:
-1. Make screenshots of your progress in the course
-   and put the screenshots in `task_offline_web_app` folder in
-   your `kottans-frontend` repo
-1. In your `kottans-frontend` repo `README.md`:
-   * add header `## Offline Web Applications`
-   * embed the screenshots or add links to the screenshots
-   * list your reflections on the course
-     (_what was new to you_, _what surprised you_, _what you intend to use in future_)
-1. You did an extra effort! Go ahead
-   share your progress with others –
-   post a message in [course channel][chat]:
-   `Offline Web Apps — #done` (or `Offline Web Apps — #p2p_done` if you are p2p course student) and add the link to your repo. **This step is important, as it helps mentors to track your progress!**
-1. Study Extra Materials below to improve your skills.
-   If you feel it affects your overall course performance consider
-   reverting to those later e.g. when you have all mandatory tasks completed.
-1. You may skip Optional materials from this task (if any).
-
-When you finish this task you can proceed to the next one.
-
-## Extra materials
+Якщо ви вирішите виконати завдання, то
+після завершення зробіть наступне:
+1. Зробіть знімок екрана своїх завершених уроків
```

Comment:
Same story as in this [comment](https://github.com/kottans/frontend/pull/361/commits/aa7b9173671e0657fb31bf90317d2cbf70f7c751)